src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -58,6 +58,13 @@
     text-overflow: ellipsis;
 }
 
+.bcs-VersionsItem-retention {
+    display: flex;
+    align-items: center;
+    margin-top: 8px;
```

Comment:
Do we need to also apply `margin-right: 35px` like `.bcs-VersionsItem-log` above to avoid flowing under the action toggle button?